src/main/java/org/wiremock/extension/jwt/JwtHelpersExtension.java

```diff
@@ -22,7 +23,23 @@ public Map<String, Helper<?>> provideTemplateHelpers() {
                 "jwks", jwksHandlebarsHelper
         );
     }
+    
+    @Override
+    public void start() {
+        initialise();
+    }
 
+    private void initialise() {
+        if (!initialised) {
```

Comment:
I am thinking we probably don't need all of this now, given start only happens once during wiremock initialisation